### PR TITLE
css-layout-classes | Richard Watanabe

### PR DESCRIPTION
**Pull request by Richard-Watanabe**
Screen Shot 2021-06-24 at 3 24 48 PM

### CODE REVIEW

File: css-layout-classes/styles.css [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+.container {
+  max-width: 800px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.column-one-third {
+  display: flex;
+  flex-direction: column;
```
Comment (codingcodymiller) [inline css-layout-classes/styles.css:18]: You can remove these, generally column classes only call for a size to be set on them, the `row` class is usually the flex container.

File: css-layout-classes/styles.css [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  width: calc(100% / 3);
+}
+
+.column-one-fourth {
+  display: flex;
+}
+
+.column-two-third {
+  display: flex;
+  flex-direction: column;
```
Comment (codingcodymiller) [inline css-layout-classes/styles.css:28]: You can remove these, generally column classes only call for a size to be set on them, the `row` class is usually the flex container.

File: css-layout-classes/styles.css [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+}
+
+.column-one-third {
+  display: flex;
+  flex-direction: column;
+  width: calc(100% / 3);
+}
+
+.column-one-fourth {
+  display: flex;
```
Comment (codingcodymiller) [inline css-layout-classes/styles.css:23]: I would change this to `width: 100%`

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Great work on this!! A couple minor revisions:

**Comment by Richard-Watanabe** [comment]
Screen Shot 2021-06-24 at 3 36 45 PM

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**